FHEM/14_SD_WS_Maverick.pm

```diff
@@ -308,26 +280,44 @@ sub SD_WS_Maverick_SetSensor2Inaktiv($){
   my $name = $hash->{NAME};
   Log3 $hash, 5, "$name SD_WS_Maverick_SetSensor2Inaktiv";
 
-  $hash->{sensor_2_state}="inactiv";
+  $hash->{sensor_2_state}="inactive";
   SD_WS_Maverick_updateReadings($hash);
 }
 
 sub SD_WS_Maverick_updateReadings($){
   my ($hash) = @_;
   my $name = $hash->{NAME};
   Log3 $hash, 5, "$name SD_WS_Maverick_updateReadings";
-  
+
   readingsBeginUpdate($hash);
     readingsBulkUpdate($hash, "temp-food", $hash->{temp_food});
```

Comment:
Irgendwie frage ich mich, wieso Du alle Daten als Internal speicherst und dann noch mal Zusätzlich als Reading.

Reply:
Diesen Code habe ich nicht geschrieben, er bestand schon vor meiner Änderung. Ich habe lediglich in der Subroutine `SD_WS_Maverick_updateReadings` Zeile 299 hinzugefügt.

Ich vermute, das ursprüngliche Autor hat die Sub `SD_WS_Maverick_updateReadings` erstellt, um Code zu vermeiden, denn sie wird von 3 Stellen aus aufgerufen wird (`Parse`, `SetSensor1Inaktiv` und `SetSensor2Inaktiv`). Anstatt die Werte als Array zu übergeben, hat er sich wohl dafür entschieden, die im Modul-Hash abzulegen.